beacon-chain/rpc/proposer_server.go

```diff
@@ -74,6 +74,18 @@ func (ps *ProposerServer) PendingAttestations(ctx context.Context, req *pb.Pendi
 	if err != nil {
 		return nil, fmt.Errorf("could not retrieve pending attestations from operations service: %v", err)
 	}
+	// Remove any attestation from the list if their slot is before the start of
+	// the previous epoch. This should be handled in the operationService cleanup
+	// method, but we should filter here in case it wasn't yet processed.
+	boundary := helpers.StartSlot(helpers.PrevEpoch(beaconState))
+	var attsInBoundary []*pbp2p.Attestation
```

Comment:
is there any advantage to `make` beforehand? ex: `make([]*pbp2p.Attestation, len(atts))

Reply:
I think so, we expect that expired attestations are rare, so I'll make this slice with an internal capacity of the length of the original set of attestations